src/sentry/web/frontend/twofactor.py

```diff
@@ -16,6 +17,7 @@
 
 
 class TwoFactorAuthView(BaseView):
+    U2F_INTERFACE = 3
```

Comment:
where does this 3 come from? is it declared anywhere else?

Reply:
by my understanding 0 is recovery codes, 1 is app, 2 is sms and 3 is u2f. they are declared in each of the separate 2fa interfaces

Reply:
okay. it's generally not best to redeclare constants, could you do `U2fInterface.type` here instead?

Reply:
yup sounds good